packages/documentation/copy/en/reference/Security.md

```diff
@@ -8,10 +8,44 @@ preamble: >
 
 The current implementation of federated execution lacks authentication of federates from joining the federation. Also communication security is not implemented to prevent adversarial federates sending malicious messages (e.g.. bad tags or bad sensor readings) to the RTI or other federates.
 
+<div class="lf-cpp lf-py lf-ts lf-rs">
+
+The $target-language$ target does not currently support the `auth` target option.
+
+</div>
+
+<div class="lf-c">
+
+## Authentication
+
 For the `C` target at least, federated execution is able to apply security with authentication by using HMAC authentication between RTI and federates. To enable this, include the `auth` property in your target specification, as follows:
 
 ```lf-c
 target C {
     auth: true
 };
-```
\ No newline at end of file
+```
+
+The RTI bui0ld must include cmake options to enable simple HMAC-based authentication of federates. Add `-DAUTH=ON` option to the cmake command as shown below:
```

Comment:
```suggestion
The RTI build must include cmake options to enable simple HMAC-based authentication of federates. Add `-DAUTH=ON` option to the cmake command as shown below:
```

Reply:
@edwardalee Thanks for checking my PR. I fixed what you mentioned on the latest commit 7972f11 . Is there any other suggestions to add or change?